### PR TITLE
app-mobilephone/gammu: remove old, add new

### PR DESCRIPTION
**Pull request by ktrace**
remove 1.38.4 (cause 1.38.5 same)
 Closes: #6936

Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

File: app-mobilephone/gammu/gammu-1.36.8.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ SRC_URI="http://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```
Comment (a17r) [inline app-mobilephone/gammu/gammu-1.36.8.ebuild]: stabilisation does not work that way, you need to file a stabilisation bug instead.

**Comment by monsieurp** [comment]
Hi @ktrace! @a17r is right. Unless you've got a good reason to mark those packages stable on your own, and you better have solid arguments, please file a stabilisation bug instead and let arch teams do the heavy lifting for you. Thanks.

**Comment by ktrace** [comment]
ok, waiting 4 merge :)

File: app-mobilephone/gammu/gammu-1.38.5.ebuild [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline app-mobilephone/gammu/gammu-1.38.5.ebuild]: I'm lost here. Why are you changing this line if you didn't modify the file?

Also this commit doesn't add anything, so please fix the commit message. Also please fix spacing between commit message lines according to GLEP66. Also you can put versions straight into summary line.

File: dev-python/python-gammu/Manifest [hunk 3]
```diff
@@ -1 +1,2 @@
+DIST python-gammu-2.11.tar.gz 135567 BLAKE2B 133172cfd0715ff6b5bd47b93e5d93d52af3e608459ba1158bf008da9b5f24f6d155c04334dea7ff2cacdea678f00d986838a139290d5f8ac874bb9702fcd5e6 SHA512 cdf3a54fc538fecdbe3dae63f0534094ce94a289998f9b670a0b3e12a3c571eb46ee86eae67a7a8d2428698850046d5c75569753bf77551cd395d315e6f9b70d
```
Comment (mgorny) [inline dev-python/python-gammu/Manifest]: Body part of this commit message doesn't add anything compared to the summary line, so drop it.

File: app-mobilephone/wammu/Manifest [hunk 4]
```diff
@@ -1,2 +1,2 @@
-DIST wammu-0.40.tar.bz2 687062 BLAKE2B 413736e40bd2a809123e176f2d00042f407c9a8bea0cc9f4a15561b7f96375c30fa1c8fc65ea8613cc7d369df13fa6841f027e69991ba323374b1dc99bf9c86d SHA512 2ed5d5cdd63b3731808c5fa8cc36caf394c4ccf714eb4ef39a25f1e7967d8ccf8b69db5aeff37ff0d1bbf7cefec51fe070a5f0b90d0cb8271f1a3d243eaf06fd
```
Comment (mgorny) [inline app-mobilephone/wammu/Manifest]: You can't remove this, it's the latest stable version.

File: app-mobilephone/gammu/Manifest [hunk 5]
```diff
@@ -1,4 +1,2 @@
-DIST gammu-1.33.0.tar.bz2 6535262 BLAKE2B 4a5c22dccfec22f1c4158ac3a98376c7c987769fb1a09e727d47ade735b2643d47a6b19aa43aa274be385fc0bbd8460443870211aee3b37bffc88780a607630b SHA512 42252a36c7ad5a2945fcf23fcb1b5bb0201d27e8830d3b4a7f4c36bf00f12f5a0f7797e17098bda8d526d4ddc3b1808c0786dfae0feef9424516fa68bf2f3f1e
```
Comment (mgorny) [inline app-mobilephone/gammu/Manifest]: Likewise, this is last stable on amd64.

File: app-mobilephone/wammu/Manifest (hunk 4 above)
Comment (ktrace) [reply app-mobilephone/wammu/Manifest]: latest stable 0.43

File: dev-python/python-gammu/Manifest (hunk 3 above)
Comment (ktrace) [reply dev-python/python-gammu/Manifest]: fixed

File: dev-python/python-gammu/python-gammu-2.11.ebuild [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+	)"
+
+DOCS=( AUTHORS NEWS.rst README.rst )
+
+python_test() {
+	esetup.py test
+}
+
+python_install_all() {
+	use examples && local EXAMPLES=( examples/. )
```
Comment (mgorny) [inline dev-python/python-gammu/python-gammu-2.11.ebuild]: Please test it with `USE=examples` enabled.

File: app-mobilephone/wammu/Manifest (hunk 4 above)
Comment (mgorny) [reply app-mobilephone/wammu/Manifest]: Even then, removal belongs in a separate commit.

Comment (ktrace) [reply app-mobilephone/wammu/Manifest]: fixed

File: dev-python/python-gammu/python-gammu-2.11.ebuild (hunk 6 above)
Comment (ktrace) [reply dev-python/python-gammu/python-gammu-2.11.ebuild]: tested. USE=test for dev-python/python-gammu require USE=dbi for gammu. with gammu[dbi] work perfect

File: app-mobilephone/gammu/gammu-1.39.0.ebuild [hunk 7]
```diff
@@ -35,10 +35,6 @@ RDEPEND="
 	dev-util/dialog
 	virtual/libiconv
 "
-PATCHES=(
-	"${FILESDIR}/${P}-smsd.patch"
-)
-
 src_configure() {
 	local mycmakeargs=(
```
Comment (SoapGentoo) [inline app-mobilephone/gammu/gammu-1.39.0.ebuild]: please change src_test not to muck with `MAKEOPTS` and instead use `LD_LIBRARY_PATH="${BUILD_DIR}/libgammu" cmake-utils_src_test -j1`

File: app-mobilephone/wammu/wammu-0.44.ebuild [hunk 8]
```diff
@@ -1,27 +1,29 @@
...
-	bluetooth? ( dev-python/pybluez[${PYTHON_USEDEP}]
+	bluetooth? (
+		dev-python/pybluez[${PYTHON_USEDEP}]
 		gnome? ( net-wireless/gnome-bluetooth )
-	)"
+	)
+"
 DEPEND="${RDEPEND}"
 
 # Supported languages and translated documentation
```
Comment (SoapGentoo) [inline app-mobilephone/wammu/wammu-0.44.ebuild]: this ebuild parses `LINGUAS`, which is bad. If the translations are installed into standard `/usr/share/locale`, then don't do this filtering and just rely on `INSTALL_MASK`

Comment (SoapGentoo) [reply app-mobilephone/wammu/wammu-0.44.ebuild]: Also, export `SKIPWXCHECK=yes` globally and don't override `distutils-r1` phase functions

File: dev-python/python-gammu/python-gammu-2.11.ebuild (hunk 6 above)
Comment (SoapGentoo) [reply dev-python/python-gammu/python-gammu-2.11.ebuild]: I also don't believe that it worked

File: dev-python/python-gammu/python-gammu-2.11.ebuild [hunk 9]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
```
Comment (SoapGentoo) [inline dev-python/python-gammu/python-gammu-2.11.ebuild]: why no python 3.6?

**Comment by mgorny** [comment]
Ping.

**Comment by ktrace** [comment]
Pong. I'm alive, but too busy, make all needed changes during week.

**Comment by mgorny** [comment]
Ok, thanks for getting back to me.

**Comment by ktrace** [comment]
Need review, seems all needed changes here.

File: app-mobilephone/gammu/gammu-1.38.4.ebuild [hunk 10]
```diff
@@ -1,63 +0,0 @@
-# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline app-mobilephone/gammu/gammu-1.38.4.ebuild]: Please update the commit message for the first commit.

1. Indicate what it does in summary.
2. Do not Close a random irrelevant pull request.

File: app-mobilephone/gammu/gammu-1.39.0.ebuild [hunk 11]
```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline app-mobilephone/gammu/gammu-1.39.0.ebuild]: Body fully repeats summary of commit message, so drop it.

File: app-mobilephone/wammu/wammu-0.40.ebuild [hunk 12]
```diff
@@ -1,52 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline app-mobilephone/wammu/wammu-0.40.ebuild]: This is in the wrong commit.

File: app-mobilephone/wammu/wammu-0.44.ebuild [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+	)
+"
+DEPEND="${RDEPEND}"
+
+# Required to source locale content out of the box
+DISTUTILS_IN_SOURCE_BUILD=1
+
+# SKIPWXCHECK: else 'import wx' results in
+# Xlib: connection to ":0.0" refused by server
+SKIPWXCHECK=yes
```
Comment (mgorny) [inline app-mobilephone/wammu/wammu-0.44.ebuild]: Please use virtualx eclass to start X server for tests.

Comment (mgorny) [reply app-mobilephone/wammu/wammu-0.44.ebuild]: Please disregard this. I misread what this does.

**Pull request closed without merging**